lib/flop_phoenix/table.ex

```diff
@@ -229,11 +231,18 @@ defmodule Flop.Phoenix.Table do
     direction = order_direction(assigns.meta.flop, assigns.field)
     assigns = assign(assigns, :order_direction, direction)
 
+    sort_path =
+      build_path(assigns[:path], assigns[:meta], assigns[:field],
+        directions: assigns[:directions]
+      )
+
+    assigns = assign(assigns, :sort_path, sort_path)
```

Comment:
I pulled this `build_path` call out of the heex template due to compilation warnings about change tracking.

Reply:
I guess because you used `assigns[:directions]` in the HEEx template? Could have been solved by setting `default: nil` on the attribute. Then you could safely use `@directions`.